### PR TITLE
HDInsight Java SDK Preview Overview

### PR DESCRIPTION
**Pull request by tylerfox**

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/hdinsight.md](https://github.com/tylerfox/azure-docs-sdk-java/blob/smoke-test/docs-ref-services/hdinsight.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/api/overview/azure/hdinsight?view=azure-java-stable&branch=pr-en-us-631) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/api/overview/azure/hdinsight?view=azure-java-preview&branch=pr-en-us-631) (azure-java-preview) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C20%5C5de83821-f44b-0e93-473e-c19393f102fe%5CPullRequest%5C201809201658443697-631%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yiJMA2R%2Bozrhd0OvTENVvBqsacp77Cajgaw7MPeqrv8%3D&st=2018-09-20T17%3A53%3A59Z&se=2018-10-21T17%3A58%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/hdinsight.md](https://github.com/tylerfox/azure-docs-sdk-java/blob/smoke-test/docs-ref-services/hdinsight.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/api/overview/azure/hdinsight?view=azure-java-stable&branch=pr-en-us-631) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/api/overview/azure/hdinsight?view=azure-java-preview&branch=pr-en-us-631) (azure-java-preview) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C20%5C5de83821-f44b-0e93-473e-c19393f102fe%5CPullRequest%5C201809201711010459-631%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=IbCFcxDNGGgAhzyfCOlbceoDFZtod2jX2DLDZnqjsmY%3D&st=2018-09-20T18%3A52%3A47Z&se=2018-10-21T18%3A57%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/hdinsight.md](https://github.com/tylerfox/azure-docs-sdk-java/blob/smoke-test/docs-ref-services/hdinsight.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/api/overview/azure/hdinsight?view=azure-java-stable&branch=pr-en-us-631) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/api/overview/azure/hdinsight?view=azure-java-preview&branch=pr-en-us-631) (azure-java-preview) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C20%5C5de83821-f44b-0e93-473e-c19393f102fe%5CPullRequest%5C201809201943472125-631%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=uJ65jWZWgGO6iE2S4M2DpXq%2BmSwFOOfZAjSuKUE79ME%3D&st=2018-09-20T20%3A39%3A51Z&se=2018-10-21T20%3A44%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/hdinsight.md [hunk 1]
```diff
@@ -1,29 +1,391 @@
...
 ## Overview
+The HDInsight Java SDK provides classes and methods that allow you to manage your HDInsight clusters. It includes operations to create, delete, update, list, scale, execute script actions, monitor, get properties of HDInsight clusters, and more.
+
+## Prerequisites
+* An Azure account. If you don't have one, [get a free trial](https://azure.microsoft.com/free/).
+* [Java JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [Maven](https://maven.apache.org/install.html)
+
+## SDK Installation
+The HDInsight Java SDK is available through Maven [here](https://mvnrepository.com/artifact/com.microsoft.azure.hdinsight.v2018_06_01_preview/azure-mgmt-hdinsight). Add the dependency to your pom.xml.
```
Comment (brunoborges) [inline docs-ref-services/hdinsight.md]: Can this be a code snippet of the `<dependency>` element for `pom.xml`? The way this paragraph is structureds, seems that the two links below are more important than the `here` hyperlink, that is more likely to be missed by the reader.

File: docs-ref-services/hdinsight.md [hunk 2]
```diff
@@ -1,29 +1,391 @@
...
+
+        HDInsightManagementClientImpl client = new HDInsightManagementClientImpl(credentials);
+```
+
+
+## Cluster Management
+> [!NOTE]
+> This section assumes you have already authenticated and constructed an `HDInsightManagementClientImpl` instance and store it in a variable called `client`. Instructions for authenticating and obtaining an `HDInsightManagementClientImpl` can be found in the Authentication section above.
+
+### Create a Cluster
```
Comment (brunoborges) [inline docs-ref-services/hdinsight.md]: It is best practice to keep elements in a Markdown with blank lines over and below, especially the titles.

As in: 

```

# Title

Text
```

Comment (tylerfox) [reply docs-ref-services/hdinsight.md]: Done, thanks

**Pull request merged**